pkg/apis/pipeline/v1alpha1/condition_validation_test.go

```diff
@@ -54,15 +54,6 @@ func TestCondition_Invalid(t *testing.T) {
 		cond          *v1alpha1.Condition
 		expectedError apis.FieldError
 	}{{
-		name: "invalid meta",
-		cond: &v1alpha1.Condition{
-			ObjectMeta: metav1.ObjectMeta{Name: "invalid.,name"},
```

Comment:
I would have expected `,` to still be invalid...

Reply:
Good catch -- I didn't notice the comma when I was removing tests. Since that's not a Tekton restriction but rather a [Kubernetes one](https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#dns-subdomain-names), do you think we should keep this test? Or potentially update the other tests to check for a comma instead of a period (using say `bad,name` instead of `bad.name`)?

Reply:
I've added back in tests to check that commas in names are invalid